.github/workflows/bump-cli.yml

```diff
@@ -0,0 +1,38 @@
+name: Bump CLI version
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+    paths:
+      - .github/actions/create-pr/action.yml
+  schedule:
+  - cron: 0 0 */14 * * # run every 14 days
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Bump CLI
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          scripts/replace-cli-version.sh
+      - name: Commit, Push and Open a PR
+        uses: ./.github/actions/create-pr
+        with:
+          token: ${{ github.token }}
```

Comment:
Also, here:

```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
I will change it 😄 Also, just wondering if there is any difference between the two? I thought the action context should make these equivalent 😅

Reply:
They are indeed functionally equivalent, according to https://docs.github.com/en/actions/security-guides/automatic-token-authentication. I personally prefer using the `secrets.` syntax since it is more obvious that it's an encrypted secret.